tool/set_sdk_ref

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash --
+#
+# Updates ref to SDK packages across pubspects. Usage:
+#
+#   tool/set_sdk_version <git ref>
+
+REF=$1
```

Comment:
Nit: if you just pass `--help` to this tool that will become the ref, which seems less than ideal?

It probably doesn't matter all that much but maybe making it a dart script and using package:args would be worth doing.